custom/icds_reports/migrations/0138_index_intermediate_tables.py

```diff
@@ -130,4 +130,47 @@ class Migration(migrations.Migration):
                 ),
             ]
         ),
+        migrations.RunSQL(
+            sql=DROP_INDEX_SQL.format(index='icds_dashboard_growth_monitoring_forms_month_59c07619'),
```

Comment:
This was created in migration 0135 when I change growth monitoring agg to be nation wide instead of state. That has since been reverted and this is not the optimal index to use for our queries